CardParts/src/Extensions/UIColor.swift

```diff
@@ -28,24 +36,91 @@ extension UIColor {
     static var Gray6 : UIColor {get{return UIColor.colorFromHex(0xF5F5F5)}}
     static var Gray7 : UIColor {get{return UIColor.colorFromHex(0xE7E7E7)}}
     static var Gray8 : UIColor {get{return UIColor.colorFromHex(0xB2B2B2)}}
+
+    //Confetti Colors
+    public static var flushOrange : UIColor {get{return UIColor.dynamicColor(light: UIColor.colorFromHex(0xFF8000), dark: UIColor.colorFromHex(0xFF9A1A))}}
+    public static var eggBlue : UIColor {get{return UIColor.dynamicColor(light: UIColor.colorFromHex(0x07C4D9), dark: UIColor.colorFromHex(0x21DEF3))}}
+    public static var blushPink: UIColor {get{return UIColor.dynamicColor(light: UIColor.colorFromHex(0xFF88EC), dark: UIColor.colorFromHex(0xFFA2FF))}}
+    public static var cerulean: UIColor {get{return UIColor.dynamicColor(light: UIColor.colorFromHex(0x0097E6), dark: UIColor.colorFromHex(0x1AB1FF))}}
+    public static var limeGreen: UIColor {get{return UIColor.dynamicColor(light: UIColor.colorFromHex(0x53B700), dark: UIColor.colorFromHex(0x6DD11A))}}
+    public static var yellowSea: UIColor {get{return UIColor.dynamicColor(light: UIColor.colorFromHex(0xFFAD00), dark: UIColor.colorFromHex(0xFFC71A))}}
+    public static var superNova: UIColor {get{return UIColor.dynamicColor(light: UIColor.colorFromHex(0xFFCA00), dark: UIColor.colorFromHex(0xFFE41A))}}
+}
+
+// MARK: - Pre-iOS 13 Compatibility
+
+extension UIColor {
+    // Adaptable Colors
+    static var SystemBlue : UIColor {get {if #available(iOS 13.0, *){return .systemBlue} else {return UIColor.color(0, green: 122, blue: 255)}}}
```

Comment:
How about `camelCasing` for color names `systemBlue` instead of `SystemBlue` ?

Reply:
I believe the reason why @lkeude96 implemented it this way is because if you do camel case such as `systemBlue` there will be a naming clash with Apples UIColor `systemBlue`

Reply:
yes precisely what @croossin mentionned. I noticed that  `Black` was previously capitalized so i used the same pattern

Reply:
I also explored adding them in a nested struct.